src/libs/utils.js

```diff
@@ -382,6 +382,55 @@ export const useConsoleAssert = (condition, message) => {
   }
 }
 
+/**
+ * Asserts that a component has an accessible label, and alerts the developer how to fix it if it doesn't.
+ *
+ * @param componentName The name of the component, which will be printed to the console if there's an alert.
+ * @param allowLabelledBy If true (default), the component can have an aria-labelledby linked to another element. Set to false to only allow aria-label.
```

Comment:
Would you mind marking the optional ones using the [jsdoc syntax](https://jsdoc.app/tags-param.html#optional-parameters-and-default-values)?